routes/index.js

```diff
@@ -196,22 +178,22 @@ router.get('/network', function(req, res) {
 
 router.get('/reward', function(req, res){
   //db.get_stats(settings.coin, function (stats) {
```

Comment:
not sure why this was ever commented out. It's from prior to your PR, so its unrelated, but i noticed it due to the spacial edits.
```suggestion
  db.get_stats(settings.coin, function (stats) {
```